recipes/default.rb

```diff
@@ -16,68 +16,148 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
 node.default['authorization']['sudo']['include_sudoers_d'] = true
 node.default['apache']['contact'] = 'hostmaster@osuosl.org'
-node.default['openstack']['compute']['config']['allow_same_net_traffic'] =
-  false
-node.default['openstack']['compute']['config']['ram_allocation_ratio'] = '5.0'
-node.default['openstack']['compute']['network']['service_type'] = 'nova'
-node.default['openstack']['compute']['network']['multi_host'] = true
-node.default['openstack']['compute']['network']['force_dhcp_release'] = true
-node.default['openstack']['libvirt']['virt_type'] = 'kvm'
-node.default['openstack']['dashboard']['keystone_default_role'] = '_member_'
-node.default['openstack']['dashboard']['ssl']['cert'] = 'horizon.pem'
-node.default['openstack']['dashboard']['ssl']['cert_url'] =
-  'file:///etc/pki/tls/certs/wildcard.pem'
-node.default['openstack']['dashboard']['ssl']['chain'] = 'wildcard-bundle.crt'
-node.default['openstack']['dashboard']['ssl']['key'] = 'horizon.key'
-node.default['openstack']['dashboard']['ssl']['key_url'] =
-  'file:///etc/pki/tls/private/wildcard.key'
-node.default['openstack']['endpoints']['compute-novnc']['scheme'] = 'https'
-node.default['openstack']['developer_mode'] = false
-node.default['openstack']['release'] = 'icehouse'
+node.default['firewall']['iptables-config'].tap do |conf|
+  conf['save_on_restart'] = 'yes'
+  conf['save_on_stop'] = 'yes'
+end
+node.default['openstack']['release'] = 'mitaka'
 node.default['openstack']['secret']['key_path'] =
   '/etc/chef/encrypted_data_bag_secret'
-node.default['openstack']['sysctl']['net.ipv4.conf.all.rp_filter'] = 0
-node.default['openstack']['sysctl']['net.ipv4.conf.default.rp_filter'] = 0
-node.default['openstack']['sysctl']['net.ipv4.ip_forward'] = 1
+# node.default['openstack']['sysctl']['net.ipv4.conf.all.rp_filter'] = 0
+# node.default['openstack']['sysctl']['net.ipv4.conf.default.rp_filter'] = 0
+# node.default['openstack']['sysctl']['net.ipv4.ip_forward'] = 1
```

Comment:
Yes, I think so as this I think this gets set in the upstream cookbooks. I'll verify that and remove if it needs to be.